extension.ts

```diff
@@ -11,38 +11,83 @@ import { showCmdLine } from './src/cmd_line/main';
 import { ModeHandler } from './src/mode/modeHandler';
 import { TaskQueue } from './src/taskQueue';
 import { Position } from './src/motion/position';
+<<<<<<< aa8f6051a847c442e374ce22920ca5bd21835aec
+=======
+
+export class EditorIdentity {
+    private _fileName: string;
```

Comment:
Yeah, filename is referring to Buffer while filename+column number is actually referring to the real editor (Tab in VS Code). We need to share almost everything (VimState, History, Register) except cursor position.

The most interesting thing is, in Vim two window reading the same buffer even shares Selection.